api/docs/release.dox

```diff
@@ -242,6 +242,10 @@ Further non-compatibility-affecting changes include:
    functionality. (Issue #3129). #module_mapper_t replaces the following
    #raw2trace_t APIs: handle_custom_data(), do_module_parsing(),
    do_module_parsing_and_mapping(), and find_mapped_trace_address().
+ - Added trace_metadata_writer_t, a set of utility functions for writing trace
+   metadata, such as process/thread ids, timestamps, etc. This unblocks moving
+   some raw2trace_t functionality into the header file (in an upcoming reusable
```

Comment:
No reason to talk about "unblocking" or about "upcoming" for immediate-future things that are all going in before any official release.  This list is for between the last official release and the next official release and is not a day-by-day append-only list.  The final entry should end up including the template stuff and any other related raw2trace interface changes probably in the same entry.  Could also combine this and the module_mapper_t in one bullet (w/ sub-bullets if nec.) about generalizing the drcachesim raw2trace interface.

Reply:
done